README.rst

```diff
@@ -93,17 +93,17 @@ The generated image looks like this:
 Full grammar
 ------------
 
-* Each text line represents either an observable, either an operator
+* Each text line represents either an observable, or an operator
 * Observables are defined with dash lines
 * If an Observable starts with a letter, this letter is considered as a label
 * Each character represent a time span
-* Character *|* indicates an observable completion
-* Character *\** indicates an observable error
-* Character *>* indicates an observable continutation
-* The *+* character is used to define anchors of higher order observables
-* operators start with a *[* and end with a *]*
+* Character ``*`` indicates an observable completion
```

Comment:
original text was wrong, but this one also: the character for completion is | (pipe).